apps/site/pages/en/about/previous-releases.mdx

```diff
@@ -50,3 +50,4 @@ Community installation methods included on the self-service download page (/down
 - **Broad OS Support:** Cannot be limited to a subset of OS distributions or versions.
   - For example, an installation method claiming compatibility with “Windows” must function on “Windows 10”, “Windows 11”, and all their editions (including server versions).
   - Similarly, an installation method claiming compatibility with “Linux” must be installable on all major Linux distributions, not just a specific subset. It cannot rely on distribution-specific package managers like `apt` or `dnf`.
+- **Free and Open Source:** Must be free to use and open source, must not be sold as a commercial product nor developed by a for-profit entity, and must not be a paid service.
```

Comment:
https://chocolatey.org/ is developed by a For-Profit entity, no?

Reply:
So I may have been hasty clicking approve. IIRC there were still some concerns from the Slack thread similar to @avivkeller's comment. There are some options developed by for-profit entity which are great value additions to the ecosystem, and specifically ones which do not have a pure open source/non-paid alternative.

I went and reviewed the slack thread and I don't think I see a *clear* conclusion to it. The main agreement was that we should use the survey to inform our decision and move the conversation onto GH. I don't believe that has happened yet. So with that, I am going to withdraw my approval for now, despite liking this direction in a general sense.

Reply:
@wesleytodd I believe the for-profit discussion was not related to this. We reached agreement that for-profit options should not be considered. Now, regarding sources such as NodeSource, we are still arguing (and this is what was not reached in an agreement) if binaries such as NodeSource's should be on the installation method dropdown. We can remove the for-profit quoting for the meantime, @MattIPv4

Reply:
Now, @wesleytodd we need to reach consensus soon, please.

Reply:
```suggestion
- **Free and Open Source:** Must be free to use and open source, must not be sold as a commercial product, and must not be a paid service.
```

Reply:
Awesome, the update addressed my point of concern. Just wanted to make sure that loose end was not accidentally being committed to in this PR. Glad y'all resolved it without waiting for me to circle back.